src/indicators/flooding/glofas/utils.py

```diff
@@ -324,16 +324,6 @@ def get_rank(observations: np.array, forecast: np.array) -> np.array:
     return rank
 
 
-def calc_mpe(observations: np.array, forecast: np.array) -> float:
```

Comment:
Removing this completely will break some GloFAS notebooks. Do you think that maybe you could add the method back, and have it call the one from `utils_general.statistics`?